clusters/build-clusters/01_cluster/openshift/ci-operator/admin_ci-operator_rbac.yaml

```diff
@@ -149,3 +149,28 @@ items:
   - kind: ServiceAccount
     name: ci-operator
     namespace: ci
+
+- kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: namespace-watcher
+  rules:
+  - apiGroups:
+    - ""
+    resources:
+    - namespaces
+    verbs:
+    - list
+    - watch
```

Comment:
I _think_  watch requires both list and get but not completely sure. Lets try it like this